modules/deleting-machine-pools.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="deleting-machine-pools{context}"]
+= Deleting a machine pool
+
+You can delete a machine pool in the event that your workload requirements have changed and your current machine pools no longer meet your needs.
+// Over time, users may find that their workload needs have changed, and may want to modify the various machine pool settings. While many of these settings can be modified, certain settings (for example, instance types and availability zones) cannot be changed once a machine pool is created. If a user finds that these settings are no longer meeting their needs, they can delete the machine pool in question and create a new machine pool with the desired settings.
+You can delete machine pools using the
+ifdef::openshift-rosa[]
+Openshift Cluster Manager or the ROSA CLI (`rosa`).
+endif::openshift-rosa[]
+ifndef::openshift-rosa[]
+Openshift Cluster Manager.
+endif::[]
+
+// Users that wish to delete the default machine pool that is automatically created during the installation of a {product-title} (ROSA) cluster can do so using the OCM or ROSA CLI.
+//
+
+ifndef::openshift-rosa[]
+.Prerequisites
+
+ifdef::openshift-rosa[]
```

Comment:
Remove lines 25-27: 

ifdef::openshift-rosa[]
* You created a ROSA cluster.
endif::openshift-rosa[]


Remove line 29: 
ifndef::openshift-rosa[]

Remove line 32: 
endif::openshift-rosa[]

Change line 45 to: 
endif::openshift-rosa[]

Reply:
Hello @bmcelvee,
   Thanks for the help in cleaning up some of these conditionals. It worked! Appreciate it. ML
CC @mjpytlak